versioned/persist/tx/src/main/java/org/projectnessie/versioned/persist/tx/TxDatabaseAdapter.java

```diff
@@ -786,7 +787,17 @@ protected Hash opLoop(
       throws VersionStoreException {
     Connection conn = borrowConnection();
 
-    try (TryLoopState tryState = newTryLoopState(retryErrorMessage, config)) {
+    try (TryLoopState tryState =
+        newTryLoopState(
+            ts ->
```

Comment:
this pattern seems to exist in up to 3 places now... should it be unified potentially?

Reply:
it can't